src/main/java/edu/umich/its/spe/SPESummary.java

```diff
@@ -66,23 +66,23 @@
 	// if using the internal wait and restart approach to cron jobs.
 
 	public void reset() {
-		startTime = LocalDateTime.now();
+		startTime = Instant.now();
```

Comment:
A comment here stating that this is in UTC would be useful

Reply:
Good idea.